packages/core-mobile/app/services/fcm/FCMService.ts

```diff
@@ -80,9 +89,12 @@ class FCMService {
         url: `${PROTOCOLS.CORE}://${ACTIONS.OpenChainPortfolio}`
       }
       await NotificationsService.displayNotification({
-        channelId: ChannelId.BALANCE_CHANGES,
+        channelId:
+          result.data.notification.android?.channelId ??
+          ChannelId.BALANCE_CHANGES,
```

Comment:
looks like you added the logic to allow undefined channelId in `NotificationsService.displayNotification`. is this defaulting to ChannelId.BALANCE_CHANGES logic still needed?

Reply:
Corrected in [Make displayNotification#channelId optional](https://github.com/ava-labs/core-mobile/pull/2126/commits/77d493028626805de126cd4ff6df054454a738d5)